lib/instrumentation/index.js

```diff
@@ -11,6 +11,7 @@ var shimmer = require('./shimmer')
 var Transaction = require('./transaction')
 
 var MODULES = [
+  '@elastic/elasticsearch',
```

Comment:
In the coming days I'll publish another module named `@elastic/elasticsearch-canary`, which is identical to `@elastic/elasticsearch`, and it will be used mainly by Kibana.
I'll ping you here as soon as it's available on npm.